crawler_mcp/crawlers/optimized/factories/content_extractor.py

```
@@ -0,0 +1,413 @@
+            if hasattr(rc, "enable_javascript"):
+                rc.enable_javascript = bool(self.config.javascript_enabled)
+            if self.config.javascript_enabled:
+                # Allow some time for hydration and DOM stabilization
+                if hasattr(rc, "wait_for_js_rendering"):
+                    rc.wait_for_js_rendering = True
+                try:
+                    rc.delay_before_return_html = max(
+                        1.0, float(rc.delay_before_return_html)
+                    )  # type: ignore[attr-defined]
+                except Exception:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)